src/pages/phonesList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getPhonesAction } from '../reducer/phones/phones';
+import Header from '../components/Header';
+import Phone from '../components/Phone';
+
+const PhonesList = () => {
+  const params = useParams();
+  const { id } = params;
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getPhonesAction(id));
+  }, [dispatch]);
+  const phonesListData = useSelector((state) => state.phones);
+  return (<div>
+
+    <Header previewLink = "/" />
+    <ul className="phones-container">
+        {
+          phonesListData.map((phoneData) => (
+          <Phone
+          id={phoneData.id}
+          key={phoneData.id}
+          name={phoneData.name}
+          image={phoneData.image} />))
+        }
+
+    </ul>
+  </div>);
+};
```

Comment:
[REQUIRED] Kindly ensure that the details page is arranged horizontally just like the one in the template with a top section containing your desired image. 
![image](https://user-images.githubusercontent.com/62938735/212150860-32bcac9b-1c2e-40fc-ad08-803025f56569.png)